src/App.js

```diff
@@ -1,20 +1,23 @@
 import React from "react";
-import logo from "./logo.png";
 import "./App.css";
+import HomePage from "./homepage";
+import Basics from "./basics";
+import Intervals from "./intervals";
+import MajorScales from "./major-scales";
+import MinorScales from "./minor-scales";
+import CircleOfFifths from "./circle-of-fifths";
 
-class App extends React.Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-        </header>
-      </div>
-    );
-  }
+function App() {
+  return (
+    <div className="App">
+      <HomePage id="Home" />
```

Comment:
HomePage seems odd naming to me for a single-page application! Maybe something like Hero, Heading, or something along these lines would fit better, depending on what the section actually contains.